python/lsst/meas/base/applyApCorr.py

```diff
@@ -253,63 +237,86 @@ def run(self, catalog, apCorrMap):
                 self.log.warning("Cannot aperture correct %s because could not find %s in apCorrMap",
                                  apCorrInfo.name, " or ".join(missingNames))
                 for source in catalog:
-                    source.set(apCorrInfo.apCorrFlagKey, True)
+                    source[apCorrInfo.apCorrFlagName] = True
```

Comment:
Oh this is horrifying.  This should not be looping over sources.  But we do need to be a bit clever because of the way that afw works (but this should work with either):
```
catalog[apCorrInfo.apCorrFlagName] = np.ones(len(catalog), dtype=np.bool_)
```
(This is the same pattern as is used below)